From linux-usb-devel-bounces@lists.sourceforge.net Tue Mar 20 04:45:56 2007
Date: Tue, 20 Mar 2007 12:45:42 +0100
From: Jon K Hellan <jon.kare.hellan@uninett.no>
Subject: USB: remove duplicated device id in airprime driver
To: linux-usb-devel@lists.sourceforge.net
Message-ID: <45FFC966.1070808@uninett.no>

From: Jon K Hellan <jon.kare.hellan@uninett.no>

Both airprime and option now want to handle vendor ID 0x1410,
device ID 0x1100. Airprime calls it 'ExpressCard34 Qualcomm 3G CDMA'.
Option calls it 'Novatel Merlin XS620/S640'. Patch attached to remove it
from airprime.

From: Jon K Hellan <jon.kare.hellan@uninett.no>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/serial/airprime.c |    1 -
 1 file changed, 1 deletion(-)

--- a/drivers/usb/serial/airprime.c
+++ b/drivers/usb/serial/airprime.c
@@ -18,7 +18,6 @@
 
 static struct usb_device_id id_table [] = {
 	{ USB_DEVICE(0x0c88, 0x17da) }, /* Kyocera Wireless KPC650/Passport */
-	{ USB_DEVICE(0x1410, 0x1100) }, /* ExpressCard34 Qualcomm 3G CDMA */
 	{ USB_DEVICE(0x413c, 0x8115) }, /* Dell Wireless HSDPA 5500 */
 	{ },
 };
